src/test/java/com/globaroman/bookshopproject/config/CustomMySqlContainer.java

```diff
@@ -0,0 +1,32 @@
+package com.globaroman.bookshopproject.config;
+
+import org.testcontainers.containers.MySQLContainer;
+
+public class CustomMySqlContainer extends MySQLContainer<CustomMySqlContainer> {
+    public static final String DB_IMAGE = "mysql:8";
+    private static CustomMySqlContainer mySqlContainer;
+
+    private CustomMySqlContainer() {
+        super(DB_IMAGE);
+    }
+
+    public static synchronized CustomMySqlContainer getInstance() {
+        if (mySqlContainer == null) {
+            mySqlContainer = new CustomMySqlContainer();
+        }
+        return mySqlContainer;
+    }
+
+    @Override
+    public void start() {
+        super.start();
+        System.setProperty("TEST_DB_URL", mySqlContainer.getJdbcUrl());
+        System.setProperty("TEST_DB_USERNAME", mySqlContainer.getUsername());
+        System.setProperty("TEST_DB_PASSWORD", mySqlContainer.getPassword());
+    }
+
+    @Override
+    public void stop() {
+    }
```

Comment:
Dont stop container? may cause timeout, or memory leak issues
```suggestion
    public void stop() {
    }
```